src/fonts.ts

```diff
@@ -11,54 +11,97 @@ const specialNames = {
 }
 
 // Nerd Font asset name
-type NerdFont = string
+interface NerdFont {
+  name: string
+  isExtra: boolean
+}
 
 interface GithubFont {
   owner: string
   repo: string
   assetStartsWith: string
   assetEndsWith: string
   staticPath: string[]
+  isExtra: boolean
```

Comment:
For what it's worth, it _might_ be cleaner to name this `isDefault` or `isRequired` so we don't need to add the property to every font. We can only add `isRequired` / `isDefault` to JetBrains Mono and assume everything else is not required if it doesn't have the property defined.